scripts/assisted_deployment.sh

```diff
@@ -6,24 +6,33 @@ function destroy_all() {
 
 function set_dns() {
     NAMESPACE_INDEX=${1:-0}
+    CLUSTER_INDEX=${2:-0}
+    NETWORK_BRIDGE=tt$NAMESPACE_INDEX
```

Comment:
Use `${..}` rather than `$..` (also in the `if` below) to make this a bit more readable